### PR TITLE
news: version 0.2

### PR DESCRIPTION
**Pull request by timeyyy**
#148

### CODE REVIEW

**Comment by justinmk** [comment]
Thanks for taking the initiative here! I started something locally but I'll just pass the notes on here. (Partial list, will update later)

- I thought it would be a good idea to have two "features" sections: "Users" and "Developers".
- development enhancements: automated static analysis with PVS studio. That means we now have static analysis by coverity, clang-scan, clang ASan/UBSan, clint, luacheck, and PVS.
- UI developers: externalized 'tabline'.
    - Every UI component that is externalized means that `nvim` can run faster.

upcoming highlights:

- floating windows: https://github.com/neovim/neovim/pull/6619
- per-window background color: https://github.com/neovim/neovim/pull/6597
- externalized cmdline: https://github.com/neovim/neovim/pull/6162
- externalized 'cursorline'/'cursorcolumn' will allow 'cursorline' without a performance penalty
- windows TUI: https://github.com/neovim/neovim/pull/6315
- windows `:terminal`: 
- pre-built package (AppImage) for Linux: https://github.com/neovim/neovim/pull/6638

File: _posts/2017-05-02-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+We are very happy to announce the release of version [0.2](https://github.com/neovim/neovim/releases/tag/v0.2.0).
+
+
+Features
+--------
+
+Here are some new developments since the last newsletter.
+
+- **Windows Support** see [Install Instructions](https://github.com/neovim/neovim/wiki/Installing-Neovim)
+        There is still on going work which can be tracked [here](https://github.com/lunixbochs/ActualVim/issues/57#issuecomment-286452725)
```
Comment (justinmk) [inline _posts/2017-05-02-newsletter.md]: actualvim URL seems out of place here

File: _posts/2017-05-02-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    d) a pile of forked libraries (neovim, msgpack, asyncio, etc) hacked up to work in the sublime environment
+
+
+We also have a bonus example from the project. The popup menu(PUM) has been recently externalized from the grid, allowing
+clients to draw the widget as they like. Actualvim has externalized the PUM in around [~100LOC](https://github.com/lunixbochs/ActualVim/commit/bd214f980688546926c17ec84418446674f62f27). A picture can be seen [here](https://github.com/lunixbochs/ActualVim/issues/57#issuecomment-286452725)
+
+### The Architecture of Neovim.
+
+Some students have studied the Architecutre of Neovim as discussed [here](https://gitter.im/neovim/neovim?at=58bc63727ceae5376a75480c)
```
Comment (justinmk) [inline _posts/2017-05-02-newsletter.md]: don't need to link to gitter now that the book is finished

Comment (timeyyy) [inline _posts/2017-05-02-newsletter.md]: Is the information correct? i.e should we link to it in our wiki ?

**Comment by timeyyy** [comment]
FYI, you can add commits to this pull request.

**Comment by timeyyy** [comment]
@justinmk 
Did Pvs studio produce any bug fixes that we could mention?

**Comment by mhinz** [comment]
Maybe it could be mentioned that timer support is now on par with Vim. I think v0.1.7 wasn't able to handle `self` in timer callbacks yet.

**Comment by mhinz** [comment]
The `DirChanged` event was added: https://github.com/neovim/neovim/pull/5928

**Comment by justinmk** [comment]
In general the features in the release notes should be checked.

File: _posts/2017-05-02-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
+---
+layout: newsletter
+title: "Newsletter #8 - Open up the Windows"
+category: newsletter
+permalink: /news/2017/05/
+---
+
+@timeyyy here representing the Neovim Team.
+
+As One of Neovim's goals is to be welcoming to Contributors, I thought i would comment on my experiences being new to the team and all. In short, It has been a Blast!
```
Comment (prabirshrestha) [inline _posts/2017-05-02-newsletter.md]: "I thought i" i should be caps.

**Comment by blueyed** [comment]
Here is a fixup: https://github.com/neovim/neovim.github.io/commit/f82901dcd41cb8a61ae481d06c95fe2a107b52ba

```diff
 _posts/2017-05-02-newsletter.md | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git i/_posts/2017-05-02-newsletter.md w/_posts/2017-05-02-newsletter.md
index c031565..6d0611a 100644
--- i/_posts/2017-05-02-newsletter.md
+++ w/_posts/2017-05-02-newsletter.md
@@ -7,7 +7,7 @@ permalink: /news/2017/05/
 
 @timeyyy here representing the Neovim Team.
 
-As One of Neovim's goals is to be welcoming to Contributors, I thought i would comment on my experiences being new to the team and all. In short, It has been a Blast!
+As one of Neovim's goals is to be welcoming to Contributors, I thought I would comment on my experiences being new to the team and all. In short, it has been a blast!
 I can strongly encourage anyone to get involved if you have been thinking about it. The community is a friendly place to be.  Come join us!
 
 We are very happy to announce the release of version [0.2](https://github.com/neovim/neovim/releases/tag/v0.2.0).
@@ -17,7 +17,7 @@ Features
 --------
 
 - **Windows Support** see [Install Instructions](https://github.com/neovim/neovim/wiki/Installing-Neovim) and [ongoing work](https://github.com/neovim/neovim/issues/5229)
-- **Binares** For windows and OSX platforms
+- **Binaries** for windows and OSX platforms
 
 
 Community Projects
@@ -29,7 +29,7 @@ Let's take a look at some projects that have sprung up recently.
 
 Now this is a cool project - Bidirectional interface between Sublimetext and Neovim. This project could be used as a reference design to port to different editors as long as you have the ability to intercept all input.
 
-The steps required to make this work (Straight from the horse's mouth)
+The steps required to make this work (straight from the horse's mouth)
 
 - sublime-facing code intercepts keys and handles reading/writing sublime state and listening for sublime events
 - neovim-facing code inputs keys and handles reading/writing vim state
@@ -41,7 +41,7 @@ clients to draw the widget as they like. Actualvim has externalized the PUM in a
 
 ### The Architecture of Neovim.
 
-Some students have studied the Architecutre of Neovim. The [results](https://delftswa.gitbooks.io/desosa-2017/content/neovim/chapter.html) look awesome.
+Some students have studied the Architecture of Neovim. The [results](https://delftswa.gitbooks.io/desosa-2017/content/neovim/chapter.html) look awesome.
 
 Development Goodies
 -------------------
@@ -75,26 +75,26 @@ to our website's [issue tracker](https://github.com/neovim/neovim.github.io).
 some events or sprints together!
 
 - @Sander2 (one of the students on the Architecture project mentioned above) has
-been experimenting with code fuzzing and successfully resulted in a [bug fix](https://github.com/neovim/neovim/pull/6557) in nvim and vim.
+been experimenting with code fuzzing which resulted in a [bug fix](https://github.com/neovim/neovim/pull/6557) in both Neovim and Vim.
 
 - @ZyX-I 's [eval.c refactor](https://github.com/neovim/neovim/pull/5119) was also merged,
-paving the way for the built-in lua for 0.3. We cannot wait!
+paving the way for the built-in Lua for 0.3. We cannot wait!
 
 - [Libuv](https://github.com/libuv/libuv/issues/1287) might be getting support
-for cygwin and other platforms. This is great new for Neovim!
+for cygwin and other platforms. This is great news for Neovim!
 
 
 Misc
 ----
 
-- [Neomutt](https://github.com/neomutt/neomutt/pull/415) may be getting a lua based testing infrastructure similar to ours.
+- [Neomutt](https://github.com/neomutt/neomutt/pull/415) may be getting a Lua based testing infrastructure similar to ours.
 
 - The [road map](https://neovim.io/roadmap/) has got your back if you need a quick overview of
 what is to come.
 
 - For a complete list of Neovim features [:help vim-differences](https://neovim.io/doc/user/vim_diff.html)
 
-- @brammool (creator of vim), as well as @chrisbra (vim contributor) have left a few comments on our issue tracker, it's nice to be working together!
+- @brammool (creator of Vim), as well as @chrisbra (Vim contributor) have left a few comments on our issue tracker, it's nice to be working together!
 
 Thank you sponsors, and thank you @justinmk @brammool et al for the awesome project.
```

**Comment by timeyyy** [comment]
>Vim. v0.1.7 
@mhinz is the version correct? Seems very old :/

**Comment by mhinz** [comment]
@timeyyy I meant that Neovim v0.2.0 is now on par with current Vim regarding timer support. The last Neovim release, v0.1.7, wasn't. :>

**Comment by justinmk** [comment]
- for UI authors: `ui_events` are now reported (and versioned) in the API metadata (`api_info()`, `--api-info`) thanks to @bfredl

**Comment by timeyyy** [comment]
http://blog.cognitect.com/blog/2017/4/17/clojure-for-neovim-for-clojure

**Pull request merged**